packages/webapp/pages/devcard.tsx

```diff
@@ -128,164 +135,200 @@ const Step2 = ({
+                Embed
+              </Button>
+
+              {selectedTab === 0 && (
+                <ActiveTabIndicator className="bottom-0 w-10" />
+              )}
+            </div>
+
+            <div className="p-2">
+              <Button
+                size={ButtonSize.Medium}
+                pressed={selectedTab === 1}
+                variant={ButtonVariant.Tertiary}
+                onClick={() => setSelectedTab(1)}
+              >
+                Customize
+              </Button>
+
+              {selectedTab === 1 && (
+                <ActiveTabIndicator className="bottom-0 w-14" />
+              )}
+            </div>
+          </div>
+
+          <div className="p-4">
+            {selectedTab === 0 && (
+              <>
+                <p className="text-theme-label-tertiary typo-callout">
+                  Show off your daily.dev activity easily by adding your DevCard
+                  to GitHub, your website, or use it as a dynamic Twitter cover
+                  (now called X, but we still like the old name better).
+                </p>
+
+                <h3 className="typo-title4 mb-2 mt-5 font-bold">
+                  <GitHubIcon className="mr-2 h-auto w-auto" /> Embed the
+                  DevCard on your GitHub profile
+                </h3>
+
+                <p className="text-theme-label-tertiary typo-callout">
+                  Find out how to put your DevCard in your GitHub README and
+                  make it update automatically using GitHub Actions. Full
+                  tutorial
+                </p>
+
+                <textarea
+                  className="mt-4 h-[7.75rem] w-full resize-none self-stretch rounded-10 bg-theme-float px-4 py-2 text-theme-label-tertiary laptopL:w-[25rem]"
+                  readOnly
+                  wrap="hard"
+                  value={embedCode}
+                />
+                <Button
+                  className="mt-4"
+                  variant={ButtonVariant.Secondary}
+                  size={ButtonSize.Small}
+                  onClick={() => copyEmbed()}
+                >
+                  {!copyingEmbed ? 'Copy code' : 'Copied!'}
+                </Button>
+
+                <h3 className="typo-title4 mb-2 mt-5 font-bold">
+                  Use as X header
+                </h3>
+                <p className="text-theme-label-tertiary typo-callout">
+                  Level up your Twitter game with a DevCard header image!
+                </p>
+              </>
+            )}
+
+            {selectedTab === 1 && (
+              <>
+                <h3 className="typo-title4 mb-2 font-bold">Theme</h3>
+
+                <div className="flex flex-row flex-wrap">
+                  {Object.keys(themeToLinearGradient).map((theme) => (
+                    <>
+                      <div
+                        className="mb-3 mr-3 h-10 w-10 rounded-full"
+                        style={{ background: themeToLinearGradient[theme] }}
```

Comment:
The themes need to be clickable and hoverable, and be available or not for use based on reputation point thresholds